### PR TITLE
fix: #1057 add align to dropdown

### PR DESCRIPTION
**Pull request by AKiroi**
image

### CODE REVIEW

File: mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
-  padding: $paddingSmall;
+  padding: $paddingSmall $paddingMedium;
+  font-size: $fontSizeMedium;
+  cursor: pointer;
 
-  &:hover {
-    background-color: var(--secondaryBgColor);
-    cursor: pointer;
+  &:not(:last-child) {
+    border-bottom: $primaryBorder solid var(--tertiaryBgColor);
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss:9]: border is not background wrong token

File: mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+  }
+  &:last-child {
+    border-bottom-left-radius: $primaryBorderRadius;
+    border-bottom-right-radius: $primaryBorderRadius;
   }
 
+  &:hover,
   &:active {
-    background-color: var(--generalPrimaryColor);
+    color: var(--selectHoverColor);
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss:22]: dropdown is not select

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss (hunk 2 above)
Comment (AKiroi) [reply mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss:22]: image

File: mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss (hunk 1 above)
Comment (AKiroi) [reply mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss:9]: image

File: mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss (hunk 2 above)
Comment (AKiroi) [reply mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss]: fixed

File: mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss (hunk 1 above)
Comment (AKiroi) [reply mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss]: fixed

File: mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
-  padding: $paddingSmall;
+  padding: $paddingSmall $paddingMedium;
+  font-size: $fontSizeMedium;
+  cursor: pointer;
 
-  &:hover {
-    background-color: var(--secondaryBgColor);
-    cursor: pointer;
+  &:not(:last-child) {
+    border-bottom: $primaryBorder solid var(--secondaryStrokeColor);
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss:9]: On the maket there are using other token-. PLease check or discuss with designers

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss (hunk 3 above)
Comment (AKiroi) [reply mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss:9]: what other token dowe need? Please, explain

Comment (AKiroi) [reply mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss]: fixed

File: mw-webapp/src/globalStore/ThemeStore.ts [hunk 4]
```diff
@@ -95,6 +95,10 @@ const themedVariables: Record<string, Record<Theme, string>> = {
     [Theme.DARK]: "#FFFFFF3E",
     [Theme.LIGHT]: "#784DEA3E",
   },
+  dropdownStrokeColor: {
+    [Theme.DARK]: "#FFFFFF3E",
```
Comment (Ekaterina1994) [inline mw-webapp/src/globalStore/ThemeStore.ts:99]: On the maket there are different value for this token

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

File: mw-webapp/src/globalStore/ThemeStore.ts (hunk 4 above)
Comment (AKiroi) [reply mw-webapp/src/globalStore/ThemeStore.ts]: fixed

**Review by Ekaterina1994 — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**